src/components/editor/use-host-bridge.js

```diff
@@ -48,10 +50,18 @@ export function useHostBridge( post, editorRef ) {
 				endComposition( editorRef.current );
 			}
 
-			return {
-				title: getEditedPostAttribute( 'title' ),
-				content: getEditedPostContent(),
-			};
+			const title = getEditedPostAttribute( 'title' );
+			const content = getEditedPostContent();
+			const changed =
+				title !== postTitleRef.current ||
+				content !== postContentRef.current;
+
+			if ( changed ) {
+				postTitleRef.current = title;
+				postContentRef.current = content;
+			}
```

Comment:
Selectively updating these refs allows us to accurately communicate a "content has changed" status to the host app, which is important for WordPress-iOS' implementation.